parser/src/keys/macos.rs

```diff
@@ -1255,6 +1275,27 @@ impl TryFrom<PageCode> for OsCode {
                 page: 0xFF,
                 code: 0x01,
             } => Ok(OsCode::KEY_SEARCH),
+            PageCode {
+                page: 0x0C,
+                code: 0x221,
+            } => Ok(OsCode::KEY_249),
+            PageCode {
+                page: 0x0C,
+                code: 0xCF,
+            } => Ok(OsCode::KEY_250),
+            PageCode {
+                // not working
```

Comment:
I was hoping to remove this comment before merging. @jtroo could you remove it?

Reply:
👌